### DISCUSSION TITLE
SSL Enabled

### DISCUSSION THREAD

**Discussion by bhavanaks** [catfan/Medoo#998] [Q&A]
Guys does this support ssl ?

	protected $option = array(
		PDO::MYSQL_ATTR_SSL_CA => '/cert/BaltimoreCyberTrustRoot.crt.pem',
	    PDO::MYSQL_ATTR_SSL_VERIFY_SERVER_CERT => false
	);

**Comment by tianyiw2013** [comment]
```php
new Medoo([
    'option' => array(
	PDO::MYSQL_ATTR_SSL_CA => '/cert/BaltimoreCyberTrustRoot.crt.pem',
        PDO::MYSQL_ATTR_SSL_VERIFY_SERVER_CERT => false
    )
]);
```
